configure.ac

```diff
@@ -1188,7 +1188,11 @@ fi
 AX_BOOST_SYSTEM
 AX_BOOST_FILESYSTEM
 AX_BOOST_THREAD
-AX_BOOST_PROCESS
+
+dnl Opt-in to boost-process
```

Comment:
This part of the OP
> ... use `./configure --without-boost-process` to opt out...

should be inverted, right?

Reply:
Fixed